homeassistant/components/custom_card.py

```diff
@@ -0,0 +1,102 @@
+"""
+Show custom ha-cards and sate-cards on Home Assistant frontend
+
+For more details about this component, please refer to the documentation
+at https://home-assistant.io/components/custom_card/
+"""
+import asyncio
+import logging
+
+import voluptuous as vol
+
+from homeassistant.helpers.entity import Entity
+from homeassistant.helpers.entity_component import EntityComponent
+import homeassistant.helpers.config_validation as cv
+
+DOMAIN = 'custom_card'
+
+ENTITY_ID_FORMAT_HA_CARD = 'custom_ha_card' + '.{}'
```

Comment:
Components are only allowed to create entities with the domain of the entity equal to the component name.

Reply:
So I should create 2 components instead? The problem is that cards on frontend are domain specific

Reply:
Hmm..  let me check the code again...